src/main/java/de/hysky/skyblocker/utils/Utils.java

```diff
@@ -524,11 +533,12 @@ private static void tickMayorCache() {
             } else {
                 LOGGER.warn("[Skyblocker] Failed to get mayor status after 5 retries! Stopping further retries until next reboot.");
             }
-            return ""; //Have to return a value for the thenAccept block.
+            return new JsonObject(); //Have to return a value for the thenAccept block.
         }).thenAccept(result -> {
             if (!result.isEmpty()) {
-                mayor = result;
-                LOGGER.info("[Skyblocker] Mayor set to {}.", mayor);
+                mayor = result.get("name").getAsString();
+                minister = result.getAsJsonObject("minister").get("name").getAsString();
+                LOGGER.info("[Skyblocker] Mayor set to {}, minister set to {}.", mayor, minister);
```

Comment:
does it need to be logged? LOGGER.debug maybe?

Reply:
I think it's nice to have it in the logs for when someone provides logs for some bug. If it's not in the logs it could help identify the cause faster, and it's only really called once per gameplay session anyway.

Reply:
I think we should look for the actual perk rather than just the mayor/minister themselves

Reply:
Marking as resolved because of [`37136d1`](https://github.com/SkyblockerMod/Skyblocker/pull/923/commits/37136d199a74d2dff5c27632170c7a3a5a2944e4)